examples/data-jpa-repository/src/main/kotlin/cnt/jpa/domain/Movie.kt

```diff
@@ -0,0 +1,16 @@
+package cnt.jpa.domain
+
+import javax.persistence.Entity
+import javax.persistence.Id
+
+@Entity
+data class Movie(
+
+    @Id
+    val id: Long,
+
+    val title: String,
+
+    val publishYear: Int
+
+)
```

Comment:
Why so many blank lines? Wouldn't
```
@Entity
data class Movie(
    @Id
    val id: Long,
    val title: String,
    val publishYear: Int
)
```
be more readable?

Reply:
i love it ;-)